queue_services/entity-emailer/pre_hook_create_database.py

```diff
@@ -0,0 +1,30 @@
+import contextlib
+import os
+import sys
+
+import sqlalchemy
+import sqlalchemy.exc
+
+from entity_emailer.config import ProdConfig
+
+DB_ADMIN_PASSWORD = os.getenv('DB_ADMIN_PASSWORD', None)
+DB_ADMIN_USERNAME = os.getenv('DB_ADMIN_USERNAME', 'postgres')
```

Comment:
Note sure about these two(`DB_ADMIN_PASSWORD`, `DB_ADMIN_USERNAME`).   Need to confirm with @rarmitag  if this needs to be updated to pull from different environment variable names.